tests/fw_rules_tests/policies/test1-scheme.yaml

```diff
@@ -1,5 +1,6 @@
 namespaceList: ../podlist/ns_list.json
 podList: ../podlist/pods_list.json
```

Comment:
podList can point to a directory with files containing both pod list and service list.

Reply:
I am not sure I understand how you want to get services from the pod list. If podList points to a directory, there may be many resource files there. We need to specify specific pods and services files.

Reply:
Let's create a new directory with only one pods list file and only one services file. If pods_list in the scheme file points to this directory, nca should parse all the relevant yaml resources from this dir.